runtime/etc/src/mli/client_runtime.c

```diff
@@ -81,30 +95,50 @@ void chpl_library_init(int argc, char** argv) {
 
   // Just hardcode these values for now.
   const char* ip = "localhost";
-  const char* setupPort = "5554";
-  const char* mainport = "5555";
-  const char* argport = "5556";
-  const char* resport = "5557";
 
   if (initialized) { return; }
   initialized = 1;
 
   // Set up the clientside ZMQ sockets.
   chpl_mli_client_init(&chpl_client);
 
-  chpl_mli_bind(chpl_client.setup_sock, ip, setupPort);
+  chpl_mli_bind(chpl_client.setup_sock, ip, "*");
 
   char* setup_sock_conn = chpl_mli_connection_info(chpl_client.setup_sock);
   chpl_mli_debugf("Setup socket used: %s\n", setup_sock_conn);
 
-  chpl_mli_client_launch(argc, argv);
-
+  // Send the setup socket as the last argument when launching the server.
+  int argcPlusSock = argc + 2;
```

Comment:
It's not that I'm "dead set" on it, but...now that it was brought up earlier through #13030, I'd kind of like us to stick to a consistent code style across these modules.

I'm not sure how egregious things are already (I'll admit, I wasn't really thinking about consistency earlier and I defaulted to `snake_case`), but I'd be willing to settle on something (strange, around these parts...) like `snake_case_for_functions` and `camelCase` for variables.

Or does it not matter? I'll defer to you.